mmv1/templates/terraform/iam/example_config_body/vertex_ai_endpoint.tf.erb

```diff
@@ -0,0 +1,3 @@
+project = google_vertex_ai_endpoint.endpoint.project
```

Comment:
nit: these lines should be indented with two spaces so that they fit in the documentation and generated test.